x-pack/test_serverless/functional/test_suites/common/discover/context_awareness/_data_source_profile.ts

```diff
@@ -10,12 +10,22 @@ import expect from '@kbn/expect';
 import type { FtrProviderContext } from '../../../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
-  const PageObjects = getPageObjects(['common', 'timePicker', 'discover', 'unifiedFieldList']);
+  const PageObjects = getPageObjects([
+    'svlCommonPage',
+    'common',
+    'timePicker',
+    'discover',
+    'unifiedFieldList',
+  ]);
   const testSubjects = getService('testSubjects');
   const dataViews = getService('dataViews');
   const dataGrid = getService('dataGrid');
 
   describe('data source profile', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.loginAsAdmin();
+    });
+
```

Comment:
I think we can remove this change as login is happening in `context_awareness/index.ts` (the file below in this diff)

Reply:
Good point, but is it possible that new test under `context_awareness` will be working with less permissions?
We were discussing to avoid (probably with lint rule) "global" login in favour of each test doing login with the minimum required permissions explicitly.

Reply:
Yes, it could be. And I don't think that the admin permission is required even now. Then we should move it from the `index` to other files in the dir.

Reply:
@wayneseymour could you give it a shot with `Editor/Developer`

Reply:
https://github.com/elastic/kibana/pull/188128/commits/77c2143b4e5ecfe33ccc698796861e68c1fee1b1